test/functional/collations.test.js

```diff
@@ -21,7 +21,7 @@ describe('Collation', function () {
     test: function () {
       const configuration = this.configuration;
       const client = configuration.newClient(`mongodb://${testContext.server.uri()}/test`);
-      const primary = [Object.assign({}, mock.DEFAULT_ISMASTER)];
+      const primary = [Object.assign({}, mock.DEFAULT_ISMASTER_36)];
```

Comment:
Just a QoL request, since this seems to be a search and replace, could we improve the naming while we're here.
If you head to `test/tools/mock.js` you can stop importing `DEFAULT_ISMASTER` altogether. And we can rename this one to be the new `DEFAULT_HELLO` or `DEFAULT_LEGACY_HELLO` or maybe just `HELLO` or `LEGACY_HELLO`, it's kind of implicitly the default.

Reply:
I've updated to use `HELLO` and `LEGACY_HELLO` now.

Reply:
(There were a few cases where the default was still used)